### PR TITLE
COMMERCE-8676 UI to show Shipment Item External Reference

### PR DESCRIPTION
**Pull request by baharturk**
https://issues.liferay.com/browse/COMMERCE-8676

from:
https://github.com/liferay-commerce/liferay-portal/pull/2757

### CODE REVIEW

**Comment by baharturk** [comment]
ci:test:sf

**Comment by baharturk** [comment]
ci:test:relevant

**Comment by baharturk** [comment]
ci:stop

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceShipments.testcase [hunk 1]
```diff
@@ -759,6 +759,274 @@ definition {
...
+			Check.checkToggleSwitch(
+				key_productSku = "MIN55861",
+				locator1 = "Checkbox#ADD_SHIPMENT_ITEMS_CHECKBOX");
+
+			Button.clickSubmitButton();
+
+			if (IsElementPresent(key_ercColumnValue = "${key_ercColumnValue}", locator1 = "CommerceOrders#SHIPMENT_ITEM_ERC_COLUMN")) {
+				AssertTextEquals(
+					locator1 = "CommerceOrders#SHIPMENT_ITEM_ERC_COLUMN",
+					value1 = "key_ercColumnValue");
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceShipments.testcase]: `${key_ercColumnValue}`

**Comment by baharturk** [comment]
ci:test:sf

**Comment by baharturk** [comment]
ci:test:relevant

**Comment by baharturk** [comment]
ci:test:commerce-functional

**Comment by baharturk** [comment]
ci:test:commerce-functional

**Comment by baharturk** [comment]
ci:forward

**Pull request closed without merging**